UI/js/Template.js

```diff
@@ -1,37 +1,37 @@
 function createdWhen(date) {
-  const when = new Date(Date.now()).getDay() - new Date(date).getDay();
-  if (when <= 10) {
-    return 'Today';
-  }
-  return `${when} days ago`;
+	const when = new Date(Date.now()).getDay() - new Date(date).getDay();
+	if (when <= 10) {
+		return 'Today';
+	}
+	return `${when} days ago`;
 }
 
 function happening(date) {
-  const monthNames = ['January', 'February', 'March', 'April', 'May', 'June',
-    'July', 'August', 'September', 'October', 'November', 'December',
-  ];
-  const d = new Date(date);
-  const day = d.getDay();
-  const month = monthNames[d.getMonth()];
-  const year = d.getFullYear();
-  const happens = `${day} / ${month} / ${year}`;
-  return `${happens}`;
+	const monthNames = ['January', 'February', 'March', 'April', 'May', 'June',
+		'July', 'August', 'September', 'October', 'November', 'December',
+	];
+	const d = new Date(date);
+	const day = d.getDay();
+	const month = monthNames[d.getMonth()];
+	const year = d.getFullYear();
+	const happens = `${day} / ${month} / ${year}`;
+	return `${happens}`;
 }
 
 function truncate(text, max) {
-  return text.substr(0, max - 1) + (text.length > max ? '&hellip;' : '');
+	return text.substr(0, max - 1) + (text.length > max ? '&hellip;' : '');
 }
 
 function loading(type) {
-  if (type === 2) {
-    return '<div class="loading"><img src="img/6.gif" alt="Loading" title="Loading" /></div>';
-  }
-  return '<div class="loading"><img src="img/5.gif" alt="Loading" title="Loading" /></div>';
+	if (type === 2) {
```

Comment:
What is type???

Reply:
i have to loading types, so type is a flag that indicates which loading type i want. i have changed the variable name now to loadingType